protocol/v2/ssv/spectest/multi_start_new_runner_duty_type.go

```diff
@@ -99,8 +99,6 @@ func (test *StartNewRunnerDutySpecTest) RunAsPartOfMultiTest(t *testing.T, logge
 	postRoot, err := test.Runner.GetRoot()
 	require.NoError(t, err)
 
-	require.EqualValues(t, test.PostDutyRunnerStateRoot, hex.EncodeToString(postRoot[:]))
```

Comment:
why was this removed?

Reply:
There's a similar check inside the if. The idea is that if `PostDutyRunnerStateRoot` differs, then we calculate `diff` and trigger `require` library providing `diff` as the reason. However, with this line it either failed without calculating diff or passed, so the if-block below was unused